fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,56 @@
+--- Supported modes:
+
+
+Martin Modes: Martin 1, Martin 2
+PD Modes: PD 50, PD 90, PD 120, PD 160, PD 180, PD 240, PD 290
+Scottie Modes: Scottie 1, Scottie 2, Scottie DX
+Robot Modes: Robot 36 Color, Robot 72 Color
+Wraase Modes: Wraase SC2 180
+
+
+The mode specifications are taken from the Dayton Paper of JL Barber:
+http://www.barberdsp.com/downloads/Dayton%20Paper.pdf
+
+
+--- Image:
+
+
+Tap "Take Picture" or "Pick Picture" menu button or
+use the Share option of any app like Gallery to load an image.
+
+
+To keep the aspect ratio, black borders will be added if necessary.
+Original image can be resend using another mode without reloading.
```

Comment:
Note that F-Droid will take this as-is, replacing each line break with a `<br>`. Hence consider putting paragraphs on one line instead of splitting it to multiple lines. Will look better that way.

Same for the other paragraphs, of course :wink: